src/commcare_cloud/fab/fabfile.py

```diff
@@ -411,6 +412,7 @@ def _deploy_without_asking(skip_record):
              "and wait for an email saying it's done. "
              "Thank you for using AWESOME DEPLOY.")
         )
+        raise
```

Comment:
Does this end up looking like a crash to the person running the deploy?

Reply:
Because if it does you could alternatively use `exit(-1)` or some such, after printing a note for the user?

Reply:
It doesn't look any more like a crash than other deploy errors (we raise those exceptions too in the next block), and as mentioned in the description I considered that but wanted it to be obvious that the deploy failed, since that is effectively the current situation, and it was missed today.